tck-dist/src/main/assembly/assembly.xml

```diff
@@ -77,7 +82,6 @@
       <includes>
         <include>jakarta.data:jakarta.data-tck</include>
         <include>jakarta.data:jakarta.data-tck:jar:sources</include>
-        <include>jakarta.data:jakarta.data-parent</include>
```

Comment:
I think you could just update this to:
```xml
<include>jakarta.data:jakarta.data-parent:pom</include>
```

Reply:
That dependency is being dropped since the pom is included specifically by the file section on lines 39-43.